pkg/config/application_ecs.go

```diff
@@ -73,6 +73,10 @@ type ECSTargetGroups struct {
 
 // ECSSyncStageOptions contains all configurable values for a ECS_SYNC stage.
 type ECSSyncStageOptions struct {
+	// Whether to delete old tasksets before creating new ones or not.
+	// If this is set, the application may be unavailable for a short of time during the deployment.
+	// Default is false.
+	Singleton bool `json:"singleton"`
```

Comment:
I feel that using `recreate` might be better than `singleton`.
 `singleton` is typically used when the desired count is strictly equal to `1`,
whereas `recreate` allows for a broader range of use cases where the desired count can take on any value.
WDYT?

Reply:
Agree, let's me adopt your idea 👍